### PR TITLE
[artifactory][artifactory-ha] value name change to correct "storageClassName"

### PR DESCRIPTION
**Pull request by Lukplop**
#### PR Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Chart Version bumped
- [x] CHANGELOG.md updated
- [x] Variables and other changes are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[artifactory]`)

**What this PR does / why we need it**:

Value name should be `storageClassName` it is mentioned few lines above in values file, if someone will not notice this litle mistake, PVC will not be assigned to target storage class 

**Special notes for your reviewer**:

### CODE REVIEW

**Comment by chukka** [comment]
I have read the CLA Document and I hereby sign the CLA #1171

**Comment by lwargin-eis** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by lwargin-eis** [comment]
recheckcla

**Comment by Lukplop** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Lukplop** [comment]
recheckcla

File: stable/artifactory-ha/templates/nginx-pvc.yaml [hunk 1]
```diff
@@ -15,11 +15,11 @@ spec:
   resources:
     requests:
       storage: {{ .Values.nginx.persistence.size | quote }}
-{{- if .Values.nginx.persistence.storageClass }}
-{{- if (eq "-" .Values.nginx.persistence.storageClass) }}
+{{- if .Values.nginx.persistence.storageClassName }}
```
Comment (danielezer) [inline stable/artifactory-ha/templates/nginx-pvc.yaml:18]: Did you change it in the values.yaml?

File: stable/artifactory-ha/CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.1.1] - Oct 06, 2020
```
Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md:4]: This is a breaking change. Please say that in the changelog. Also, this justifies a minor version bump IMO

File: stable/artifactory-ha/templates/nginx-pvc.yaml (hunk 1 above)
Comment (chukka) [reply stable/artifactory-ha/templates/nginx-pvc.yaml:18]: @danielezer  its already corrected in values.yaml

File: stable/artifactory-ha/CHANGELOG.md (hunk 2 above)
Comment (chukka) [reply stable/artifactory-ha/CHANGELOG.md:4]: will update

Comment (chukka) [reply stable/artifactory-ha/CHANGELOG.md]: Done

File: stable/artifactory-ha/CHANGELOG.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 # JFrog Artifactory-ha Chart Changelog
 All changes to this chart will be documented in this file.
 
+## [4.2.0] - Oct 06, 2020
```
Comment (danielezer) [inline stable/artifactory-ha/CHANGELOG.md:4]: @chukka Please indicate that this is a breaking change in the changelog

**Review by chukka — APPROVED** [review]

**Review by danielezer — APPROVED** [review]

**Review by rahulsadanandan — APPROVED** [review]

**Pull request merged**